packages/cli/lib/index.js

```diff
@@ -31,7 +31,8 @@ prog
 	.option('--analyze', 'Launch interactive Analyzer to inspect production bundle(s)')
 	.option('--prerenderUrls', 'Path to pre-rendered routes config', 'prerender-urls.json')
 	.option('-c, --config', 'Path to custom CLI config', 'preact.config.js')
-	.option('--esm', '[EXPERIMENTAL] Builds ES-2015 bundles for your code.', false)
+	.option('--esm', 'Builds ES-2015 bundles for your code.', false)
+	.option('--critical-css', 'Adds critical cssto the prerendered markup.', true)
```

Comment:
I would make this `--inline` or `--inline-css`, but maybe that's just me /shrug

Reply:
Nah that sounds better